### ISSUE TITLE
Email upon submitting for review

### ISSUE THREAD

**Issue by jmkoni** [rubyforgood/new_sanctuary_asylum#43]
In #8, we had this functionality requested:
Send an email notification as follows:
If the friend has been assigned to a lawyer, email the lawyer to let them know:
"[Friend first name]'s [application category] application draft has been submitted for review."
If the friend has not been assigned to a lawyer, email the regional admins:
"[Friend first name]'s [application category] application draft has been submitted for review."
**We will change these emails a bit once the lawyer functionality has been built out

Due to time constraints, we weren't able to get to it.

**Comment by latazzajones** [comment]
I started to pencil this in here: https://github.com/rubyforgood/new_sanctuary_asylum/compare/submitting-for-review...rubyforgood:email-when-submitting-review?expand=1

But this story can't be completed until this story about lawyers being assigned friends is complete: https://github.com/rubyforgood/new_sanctuary_asylum/issues/16

If someone other than me picks this up - feel free build on that branch or start from scratch as you choose.

**Comment by CZagrobelny** [comment]
@jmkoni @latazzajones thanks for moving this forward!  I just merged the data/model changes for the remote clinic lawyer assignment.  The seed file has been updated to include some remote clinic lawyer associations and this method will retrieve all remote_clinic_lawyer users for a friend: https://github.com/rubyforgood/new_sanctuary_asylum/blob/master/app/models/friend.rb#L82
I believe this should clear the way for Tasha to complete the email functionality.  Thanks!!!
